programs/lz4cli.c

```diff
@@ -363,6 +379,23 @@ int main(int argc, const char** argv)
                 if (!strcmp(argument,  "--help")) { usage_advanced(exeName); goto _cleanup; }
                 if (!strcmp(argument,  "--keep")) { LZ4IO_setRemoveSrcFile(0); continue; }   /* keep source file (default) */
                 if (!strcmp(argument,  "--rm")) { LZ4IO_setRemoveSrcFile(1); continue; }
+                if (longCommandWArg(&argument, "--fast")) {
+                        /* Parse optional window log */
```

Comment:
This isn't a window log, it is the "acceleration factor".

Reply:
I just updated this comment in lz4. Since I copied this line from zstd, should I change it in zstd as well?

Reply:
Yeah, that'd be great!